auto-random.md

```diff
@@ -142,8 +142,9 @@ This attribute supports forward compatibility, namely, downgrade compatibility.
 
 Pay attention to the following restrictions when you use `AUTO_RANDOM`:
 
-- Specify this attribute for the primary key column **ONLY** of integer type. Otherwise, an error might occur. In addition, when the attribute of the primary key is `NONCLUSTERED`, `AUTO_RANDOM` is not supported even on the integer primary key. For more details about the primary key of the `CLUSTERED` type, refer to [clustered index](/clustered-indexes.md).
+- Specify this attribute for the primary key column **ONLY** of `bigint` type. Otherwise, an error might occur. In addition, when the attribute of the primary key is `NONCLUSTERED`, `AUTO_RANDOM` is not supported even on the integer primary key. For more details about the primary key of the `CLUSTERED` type, refer to [clustered index](/clustered-indexes.md).
 - You cannot use `ALTER TABLE` to modify the `AUTO_RANDOM` attribute, including adding or removing this attribute.
+- You cannot use `ALTER TABLE` to changing from `AUTO_INCREMENT` to `AUTO_RANDOM` if the maximum value is close to the maximum value of the column type.
```

Comment:
I have not tested, but I think the technical issue is when the maximum value is close to use the shard bit, see https://github.com/pingcap/tidb/pull/20512#discussion_r593060069 so the more shard bits used, the lower the max auto_increment.

Reply:
Is this ok as is or do I need to change the doc?

Reply:
```suggestion
- You cannot use `ALTER TABLE` to change a column from `AUTO_INCREMENT` to `AUTO_RANDOM` if the maximum value of the column is close to the maximum value of the column type.
```

Reply:
I think this is generic enough for the documentation.